eng/common/TestResources/New-TestResources.ps1

```diff
@@ -611,8 +614,14 @@ try {
         }
     }
 
+
+    if ($UseUserCredentials) {
```

Comment:
should we error out if `$TestApplicationId` is populated and `$UseUserCredentials` is true?

Reply:
I think we could just log a warning